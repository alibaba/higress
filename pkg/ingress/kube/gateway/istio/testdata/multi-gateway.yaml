apiVersion: gateway.networking.k8s.io/v1alpha2
kind: GatewayClass
metadata:
  name: higress
spec:
  controllerName: higress.io/gateway-controller
---
apiVersion: gateway.networking.k8s.io/v1alpha2
kind: Gateway
metadata:
  name: gateway
  namespace: higress-system
spec:
  gatewayClassName: higress
  addresses:
  - type: Hostname
    value: higress-gateway
  - type: Hostname
    value: higress-gateway.not-default.svc.domain.suffix
  - type: Hostname
    value: example.com
  listeners:
  - name: http
    hostname: "*.domain.example"
    port: 80
    protocol: HTTP
  - name: tcp
    port: 34000
    protocol: TCP
    allowedRoutes:
      namespaces:
        from: All
