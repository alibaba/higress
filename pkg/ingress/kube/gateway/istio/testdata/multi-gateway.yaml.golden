apiVersion: networking.istio.io/v1alpha3
kind: Gateway
metadata:
  annotations:
    internal.istio.io/gateway-service: higress-gateway.higress-system.svc.domain.suffix,higress-gateway.not-default.svc.domain.suffix,example.com
    internal.istio.io/parents: Gateway/gateway/http.higress-system
  creationTimestamp: null
  name: gateway-istio-autogenerated-k8s-gateway-http
  namespace: higress-system
spec:
  servers:
  - hosts:
    - higress-system/*.domain.example
    port:
      name: default
      number: 80
      protocol: HTTP
---
apiVersion: networking.istio.io/v1alpha3
kind: Gateway
metadata:
  annotations:
    internal.istio.io/gateway-service: higress-gateway.higress-system.svc.domain.suffix,higress-gateway.not-default.svc.domain.suffix,example.com
    internal.istio.io/parents: Gateway/gateway/tcp.higress-system
  creationTimestamp: null
  name: gateway-istio-autogenerated-k8s-gateway-tcp
  namespace: higress-system
spec:
  servers:
  - hosts:
    - '*/*'
    port:
      name: default
      number: 34000
      protocol: TCP
---
